webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -146,8 +156,11 @@ const ExperimentSingleTrialTabs: React.FC<Props> = ({ experiment, onTrialLoad }:
   }, [ fetchTrialDetails, prevTrialId, trialId ]);
 
   return (
-    <>
-      <Tabs defaultActiveKey={tabKey} onChange={handleTabChange}>
+    <TrialLogPreview
```

Comment:
I actually started out with the sibling approach but decided to switch to the parent approach to wrap around the tabs in case we want to use this preview elsewhere. The parent approach provided more control over how the preview can be laid out relative to the content he needs to "hover" over. As a sibling to the content it can hover over, it requires modification of the container of both the preview + content to support it. This brings that modification in-house into the preview component.

Reply:
chatted offline about this